doc/releases/migration-guide-3.6.rst

```diff
@@ -30,7 +30,7 @@ anymore:
 
 To enable them again use the ``west config manifest.project-filter -- +<module
 name>`` command, or ``west config manifest.group-filter -- +optional`` to
-enable all optional modules, and then run ``west update`` again.
+enable all optional modules, and then run ``west update`` again. (:github:`64139`)
```

Comment:
This reference should go at the end of the line on the canopennode" bullet instead.

Reply:
updated, is that what you meant?

Reply:
Exactly, thanks.